tests/test_software_upgrade.sh

```diff
@@ -73,7 +73,7 @@ if [ -n "$upgrade_name" ]; then
     txhash=$($proposal | jq -r .txhash)
 
     # Wait for the proposal to go on chain
-    sleep 6
+    sleep 12
```

Comment:
For stateful tests it fail sometimes due to block timeout so it cannot get the proposal ID,

Reply:
> What do you think about adding a badge for the export-mainnet-genesis workflow?

I thought of that but since this is done with an external VM instead of Github Action there won't be a way to show a badge if the test failed or pass.

Reply:
I got a way to have Github action to parse the files in the `logs` directory which will create a badge.
https://github.com/hyphacoop/cosmos-ansible/actions/workflows/export-mainnet-status-badge.yml